### ISSUE TITLE
Problem with the identifier field

### ISSUE THREAD

**Issue by AntonPetrov** [BGSU-RNA/JAR3D#11]
Command:
```
java -jar jar3d_2014-12-11.jar IL/1.18/lib/IL_85647.7.fasta IL/1.18/lib/all.txt output.loops output.txt
```

Output:
```
filename,identifier,motifId,passedCutoff,meanCutoffScore,score,deficit,interiorEditDistance,fullEditDistance,rotation
IL/1.18/lib/IL_85647.7.fasta, IL_85647.7 IL_1Q96_001 C    6 G   24,IL_00346.1,false,-152.47567657514952,-44.403,30.119,11,14,1.0000
```

What is the meaning of this text: "IL_85647.7 IL_1Q96_001 C    6 G   24"?

**Comment by clzirbel** [comment]
These are names for each sequence in the file IL_85647.7.fasta.  In order, they are the motif group id, the loop id, and the starting nucleotide on each strand of the internal loop.

**Comment by AntonPetrov** [comment]
My feeling is that this output is going to be very hard to parse. If all this data is really necessary, it should be split into separate fields and properly documented. It's not a good practice to merge things into a single field like this.

**Comment by clzirbel** [comment]
This is not output of the .jar file, it is just passing along data in 
the fasta file.  That data is written out when the motif groups are made 
by Matlab code.  It has to fit on a single line and there is no point 
making it super verbose.  It's not really meant for people to have to read.

**Comment by AntonPetrov** [comment]
OK, got it, this string indeed comes from the input file, so all is well. Sorry for the panic!
